docker/odyssey.conf

```diff
@@ -3,7 +3,7 @@ storage "postgres_server" {
 	type "remote"
 	
     host "localhost"
-	port 5432
```

Comment:
changes in config-examples is ok, anyway we need to provide more simple and straightforward config examples, but is that really needed?